client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -88,7 +88,7 @@ export class CommentDetailReply extends Component {
 							onChange={ this.handleTextChange }
 							onFocus={ this.setFocus }
 							onKeyDown={ this.submitCommentOnCtrlEnter }
-							placeholder={ translate( 'Enter your comment here…' ) }
+							placeholder={ translate( 'Reply to comment…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **95** times:
`translate( 'Reply to Comment' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments-tweaks)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).